response.go

```diff
@@ -218,13 +218,30 @@ func (i *Inertia) htmlContainer(pageJSON []byte) (inertia, _ template.HTML, _ er
 // Otherwise, it will do an HTTP redirect with specified status (default is 302 for GET, 303 for POST/PUT/PATCH).
 func (i *Inertia) Location(w http.ResponseWriter, r *http.Request, url string, status ...int) {
 	if IsInertiaRequest(r) {
+		i.captureValidationErrors(r)
 		setInertiaLocationInResponse(w, url)
 		return
 	}
 
 	redirectResponse(w, r, url, status...)
 }
 
+func (i *Inertia) captureValidationErrors(r *http.Request) {
```

Comment:
Right now it returns early if there is no errorStore. Maybe it's smart to still check and add the errors from context before checking if there is an errorStore

Reply:
Sorry, I rethought some ideas and it seemed easier to start over.

Reply:
@romsar I just saw, I think that is the correct way, not forcing people to use it but add the possibility.